sechub-cli/src/daimler.com/sechub/cli/job.go

```diff
@@ -103,46 +104,43 @@ func getSecHubJobState(context *Context, checkOnlyOnce bool, checkTrafficLight b
 		} else {
 			// PROGRESS bar ... 50 chars with dot, then next line...
 			if newLine {
-				fmt.Print("                             ")
+				sechubUtil.PrintIfNotSilent(strings.Repeat(" ", 29), context.config.quiet)
 				newLine = false
 			}
-			fmt.Print(".")
+			sechubUtil.PrintIfNotSilent(".", context.config.quiet)
 			if cursor++; cursor == 50 {
-				fmt.Println()
+				sechubUtil.PrintIfNotSilent("\n", context.config.quiet)
 				cursor = 0
 				newLine = true
 			}
 			time.Sleep(time.Duration(context.config.waitNanoseconds))
 		}
 	}
-	fmt.Print("\n")
+	sechubUtil.PrintIfNotSilent("\n", context.config.quiet)
 
 	if downloadReport {
 		downloadSechubReport(context)
 	}
 
-	/* FAIL mode */
-	if status.TrafficLight == "" {
-		fmt.Println("  No traffic light available! Seems job has been broken.")
-		os.Exit(ExitCodeFailed)
-	}
-	if status.TrafficLight == "RED" {
+	/* Evaluate traffic light */
+	switch status.TrafficLight {
+	case "RED":
 		fmt.Println("  RED alert - security vulnerabilities identified (critical or high)")
 		os.Exit(ExitCodeFailed)
-	}
-	if status.TrafficLight == "YELLOW" {
+	case "YELLOW":
 		fmt.Println("  YELLOW alert - security vulnerabilities identified (but not critical or high)")
```

Comment:
when stop on yellow == true -> print out at error stream (because it's like RED...)

Reply:
done